apps/test/unit/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditorTest.jsx

```diff
@@ -26,17 +48,30 @@ describe('ProgrammingExpressionsEditor', () => {
         }
       ],
       programmingEnvironments: [],
+      addProgrammingExpression,
       removeProgrammingExpression
     };
   });
 
+  afterEach(() => {
+    restoreRedux();
+  });
+
   it('renders default props', () => {
-    const wrapper = mount(<ProgrammingExpressionsEditor {...defaultProps} />);
+    const wrapper = mount(
```

Comment:
Why do you need `mount` here instead of `shallow`?

Reply:
It couldn't check for parts of the table without mount

Reply:
you shouldn't need to check for parts of the table; that kind of check should be handled by tests on the component that actually renders the table; the most these tests should do is check for that component

Reply:
Totally makes sense. Those tests were existing tests so I'm going to lean in favor of making them work for now and trying to get this in.